specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2021-08-01/managedClusters.json

```diff
@@ -4908,7 +4908,7 @@
           }
         ]
       },
-      "description": "Determines the placement of emptyDir volumes, container runtime data root, and Kubelet ephemeral storage."
+      "description": "Determines the workload types a node can run, currently OCI containers with Kubelet or WASM workloads with Krustlet and wasmtime."
```

Comment:
minor: I'm not sure I'd bother calling out the two enum options here. You already described that on the type itself and in most contexts, it's very easy to get to that from the property.

The reason I say this is just that documentation that is fragile to future changes (enum additions in this case) end up being missed and then being incorrect in the future.

Reply:
```suggestion
      "description": "Determines the type of workload a node can run."
```

Reply:
just one, yep. thanks!